ts/Stock/Navigator/Navigator.ts

```diff
@@ -1274,8 +1275,8 @@ class Navigator {
                 ordinal: baseXaxis.options.ordinal,
                 overscroll: baseXaxis.options.overscroll
             }, navigatorOptions.xAxis, {
-                id: 'navigator-x-axis',
-                yAxis: 'navigator-y-axis',
+                id: navigatorOptions.xAxis?.id,
+                yAxis: navigatorOptions.yAxis?.id,
```

Comment:
Do we need to define it here, since it's defined in default options? Perhaps we can remove these two lines and it should be fine. 

Interesting fact: there is no `yAxis` option for `xAxis`, so I guess that's a mistake anyway? 🤔